src/Bicep.MSBuild/build/Azure.Bicep.MSBuild.targets

```diff
@@ -38,9 +60,19 @@
     </ItemGroup>
   </Target>
 
+  <Target Name="BicepParamCompile" Inputs="@(_BicepParamResolved);@(_BicepParamSkipped)" Outputs="%(_BicepParamResolved.OutputFile)" DependsOnTargets="$(BicepCompileDependsOn)" AfterTargets="$(BicepCompileAfterTargets)" BeforeTargets="$(BicepCompileBeforeTargets)">
```

Comment:
I've reused the same `BicepCompileAfterTargets` & `BicepCompileBeforeTargets` - wasn't sure whether that's a good practice or they should be duplicated.

Reply:
Should probably separate them.